Include/cpython/pyerrors.h

```diff
@@ -82,6 +82,13 @@ typedef struct {
     PyObject *name;
 } PyAttributeErrorObject;
 
+typedef struct {
+    PyException_HEAD
+    PyObject *msg;
+    PyObject *iterable;
+    PyObject *expected_count;
```

Comment:
It would be good document those attributes. See https://docs.python.org/3/library/exceptions.html#OSError. (Namely, we need a real entry for the error not just the error being listed). I'm also not convinced by keeping a reference to the iterable in the exception (see my comments in ceval.c)